### PR TITLE
Fix the content switcher to display clients information enrolled only on HIV program

### PR DESCRIPTION
**Pull request by Daphne210**
Screenshot (294)

https://metsprogramme.atlassian.net/browse/U4X-461

### CODE REVIEW

File: packages/esm-care-panel-app/src/care-panel/care-panel.component.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ const CarePanel: React.FC<CarePanelProps> = ({ patientUuid, formEntrySub, launch
           </div>
         </CardHeader>
         <div style={{ width: '100%', minHeight: '20rem' }}>
-          {switchItem?.name === 'HIV Program' ? (
+          {programEnrolled === 'HIV Program' ? (
             <>
```
Comment (jabahum) [inline packages/esm-care-panel-app/src/care-panel/care-panel.component.tsx:69]: we dont need this extra fragement

Comment (Daphne210) [reply packages/esm-care-panel-app/src/care-panel/care-panel.component.tsx]: @jabahum This has been adjust in the preceding comment

Comment (slubwama) [reply packages/esm-care-panel-app/src/care-panel/care-panel.component.tsx]: What about TB

**Review by akileng56 — APPROVED** [review]

File: packages/esm-care-panel-app/src/care-panel/care-panel.component.tsx [hunk 2]
```diff
@@ -65,17 +65,15 @@ const CarePanel: React.FC<CarePanelProps> = ({ patientUuid, formEntrySub, launch
...
-              <ProgramEnrollment
-                patientUuid={patientUuid}
-                programName={switchItem?.name}
-                enrollments={enrollments}
-                formEntrySub={formEntrySub}
-                launchPatientWorkspace={launchPatientWorkspace}
-                PatientChartProps={''}
-              />
-            </>
+          {programEnrolled === 'HIV Program' ? (
```
Comment (akileng56) [inline packages/esm-care-panel-app/src/care-panel/care-panel.component.tsx:68]: Probably add something for the other programs is there's no data on them.. `No data for selected program`

**Review by akileng56 — CHANGES_REQUESTED** [review]

**Review by Mwanje — APPROVED** [review]

**Review by akileng56 — APPROVED** [review]

**Pull request merged**